compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/model/types/BAnyType.java

```diff
@@ -23,38 +23,40 @@
 import org.ballerinalang.model.Name;
 import org.ballerinalang.model.types.SelectivelyImmutableReferenceType;
 import org.ballerinalang.model.types.TypeKind;
-import org.wso2.ballerinalang.compiler.semantics.analyzer.SemTypeHelper;
 import org.wso2.ballerinalang.compiler.semantics.model.TypeVisitor;
 import org.wso2.ballerinalang.compiler.semantics.model.symbols.BTypeSymbol;
 import org.wso2.ballerinalang.compiler.semantics.model.symbols.Symbols;
 import org.wso2.ballerinalang.compiler.util.TypeTags;
 import org.wso2.ballerinalang.util.Flags;
 
+import static io.ballerina.types.PredefinedType.IMPLEMENTED_TYPES;
+import static io.ballerina.types.PredefinedType.VAL_READONLY;
+
 /**
  * @since 0.94
  */
 public class BAnyType extends BBuiltInRefType implements SelectivelyImmutableReferenceType {
 
     public BAnyType(BTypeSymbol tsymbol) {
-        this(tsymbol, SemTypeHelper.READONLY_SEMTYPE);
+        this(tsymbol, VAL_READONLY);
```

Comment:
Previously `READONLY_SEMTYPE` was used because the semtype component for this was same as the readonly type's semtype component. However, now the semtype component is a BasicTypeBitSet with previous bits plus the new list bit.

Reply:
Fixed